modules/ROOT/pages/rest-sdk/edits/project-setup.adoc

```diff
@@ -0,0 +1,62 @@
+= Setup and Installation
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+To get started with REST SDK, ensure you meet the prerequisites and then follow the
+steps in this section to install the REST SDK plugin.
+
+== Prerequisites
+
+Before you start using REST SDK, verify that you have the tools needed:
+
+* Java 8 SDK
+* Apache Maven (3.3.9 or newer recommended)
+
+== Install the REST SDK Plugin
+
+Install the https://repository-master.mulesoft.org/nexus/service/local/repositories/releases/content/com/mulesoft/connectors/rest-connect-cli/1.0.0-RC2/rest-connect-cli-1.0.0-RC2-jar-with-dependencies.jar[REST SDK plugin] to use REST SDK. The plugin is a distribution
```

Comment:
Because this is a single-step task topic, I suggest isolating the user action from the conceptual information:
```suggestion
Install the https://repository-master.mulesoft.org/nexus/service/local/repositories/releases/content/com/mulesoft/connectors/rest-connect-cli/1.0.0-RC2/rest-connect-cli-1.0.0-RC2-jar-with-dependencies.jar[REST SDK plugin] to use REST SDK. 
+
The plugin is a distribution
```